src/app/components/applications/SSV/MyAccount/components/Validator/BulkActions/BulkComponent.tsx

```diff
@@ -82,20 +82,24 @@ const BulkComponent = () => {
     const selectedValidatorKeys =  Object.keys(selectedValidators);
     const selectedValidatorValues =  Object.values(selectedValidators);
     let res;
-    const condition = selectedValidatorValues.every((validator: { validator: IValidator, isSelected: boolean }) => validator.isSelected);
+    const condition = selectedValidatorValues.every((validator: BulkValidatorData) => validator.isSelected);
     if (currentStep === BULK_STEPS.BULK_ACTIONS) {
       setCurrentStep(BULK_STEPS.BULK_CONFIRMATION);
     } else if (currentStep === BULK_STEPS.BULK_CONFIRMATION && currentBulkFlow === BULK_FLOWS.BULK_EXIT) {
       const singleFormattedPublicKey = formatValidatorPublicKey(selectedValidatorKeys[0]);
-      res = await conditionalExecutor(condition, async () => await validatorStore.exitValidator(singleFormattedPublicKey, process.item.operators.map((operator: IOperator) => operator.id)), async () => await validatorStore.bulkExitValidators(selectedValidatorKeys.filter((publicKey: string) => selectedValidators[publicKey].isSelected), process.item.operators.map((operator: IOperator) => operator.id)));
+      const exitSingle = async () => await validatorStore.exitValidator(singleFormattedPublicKey, process.item.operators.map((operator: IOperator) => operator.id));
+      const exitBulk = async () => await validatorStore.bulkExitValidators(selectedValidatorKeys.filter((publicKey: string) => selectedValidators[publicKey].isSelected), process.item.operators.map((operator: IOperator) => operator.id));
+      res = await conditionalExecutor(condition, exitSingle, exitBulk);
```

Comment:
don't use this function - it does not simplify anything but makes code less readable

Reply:
done